packages/starlight/components/MobileMenuFooter.astro

```diff
@@ -1,17 +1,30 @@
 ---
-import { LanguageSelect, ThemeSelect } from 'virtual:starlight/components';
+import { LanguageSelect, ThemeSelect, SocialIcons } from 'virtual:starlight/components';
 import type { Props } from '../props';
 ---
 
 <div class="mobile-preferences sl-flex">
+	<div class="sl-flex social-icons">
+		<SocialIcons {...Astro.props} />
+	</div>
 	<ThemeSelect {...Astro.props} />
 	<LanguageSelect {...Astro.props} />
 </div>
 
 <style>
+	.social-icons {
+		margin-inline-end: auto;
+		gap: 1rem;
+		align-items: center;
```

Comment:
Looking good!

I noticed one issue when it wraps (thanks for sharing the screenshot):

<img width="277" alt="social icons, language and theme selectors inside Starlight’s mobile menu" src="https://github.com/withastro/starlight/assets/357379/2ea76e21-5ace-4df0-bb17-2bb08eb5be10">

This is because when inline with other items, we center the icons but they’re actually not as tall as the select menus so the line collapses in height when they’re the only thing there. Not 100% sure whether this is the correct fix but here’s a quick way to solve it:

```suggestion
		align-items: center;
		padding-block: 1rem;
```